cpp/oneapi/dal/algo/subgraph_isomorphism/backend/cpu/graph.hpp

```diff
@@ -205,7 +206,9 @@ void graph<Cpu>::init_list_representation(const dal::preview::detail::topology<s
     bool has_edges_attribute = false;
     std::int64_t vertex_count = t._vertex_count;
     graph_data<Cpu> graph_data_storage;
-    graph_data_storage.plist_data = new graph_input_list_data<Cpu>(vertex_count, allocator_);
+    graph_input_list_data<Cpu>* memory = allocator_.allocate<graph_input_list_data<Cpu>>(1);
+    graph_data_storage.plist_data =
```

Comment:
If user's allocator or corresponding allocator_traits have 'construct' member function, they both to be ignored.